client/src/main/java/io/split/client/events/EventsSender.java

```diff
@@ -18,20 +18,20 @@ public class EventsSender {
 
     private static final String BULK_ENDPOINT_PATH = "api/events/bulk";
     private final URI _bulkEndpoint;
-    private final CloseableHttpClient _client;
+    private final SplitHttpClient _client;
     private final TelemetryRuntimeProducer _telemetryRuntimeProducer;
```

Comment:
```suggestion
```

Reply:
When I remove this one I get error for line 33:
_telemetryRuntimeProducer = checkNotNull(telemetryRuntimeProducer);

Reply:
ok, it'll be a change for the future to remove from the constructor parameters.